### PR TITLE
Sections examples, graphql-api, query-squid replaced with develop

### PR DESCRIPTION
**Pull request by abernatskiy**
deleted:    docs/examples.md
	new file:   docs/examples/_category_.json
	new file:   docs/examples/evm/_category_.json
	new file:   docs/examples/evm/evm-logs-example.md
	new file:   docs/examples/evm/factory-example.md
	new file:   docs/examples/evm/list.md
	new file:   docs/examples/evm/multicall-example.md
	new file:   docs/examples/substrate/_category_.json
	new file:   docs/examples/substrate/balances-squid.md
	new file:   docs/examples/substrate/list.md
	modified:   docs/faq.md
	new file:   docs/graphql-api/_category_.json
	modified:   docs/graphql-api/caching.md
	modified:   docs/graphql-api/custom-resolvers.md
	modified:   docs/graphql-api/dos-protection.md
	deleted:    docs/graphql-api/index.md
	new file:   docs/graphql-api/overview.md
	modified:   docs/graphql-api/subscriptions.md
	new file:   docs/query-squid/_category_.json
	modified:   docs/query-squid/and-or-filters.md
	modified:   docs/query-squid/cross-relation-field-queries.md
	deleted:    docs/query-squid/index.md
	new file:   docs/query-squid/intro.md
	modified:   docs/query-squid/json-queries.md
	modified:   docs/query-squid/nested-field-queries.md
	modified:   docs/query-squid/paginate-query-results.md
	modified:   docs/query-squid/queries.md
	modified:   docs/query-squid/resolve-union-types-interfaces.md
	modified:   docs/query-squid/sorting.md
	modified:   docs/substrate-indexing/batch-processor-in-action.md

### CODE REVIEW

File: docs/graphql-api/subscriptions.md [hunk 1]
```diff
@@ -1,30 +1,32 @@
...
+description: Subscribe to updates over a websocket
 ---
 
 # Query subscriptions
 
 **Available since `@subsquid/graphql-server@2.0.0`**
 
 :::info
-This is an experimental feature. Reach out on [Squid Devs Chat](https://t.me/HydraDevs) for more details.
+This is an experimental feature. Reach out to us at [Squid Devs Chat](https://t.me/HydraDevs) for more details.
```
[the remark this answers is not in the document]
Comment (dzhelezov) [reply docs/graphql-api/subscriptions.md:12]: @abernatskiy

**Review by dzhelezov — APPROVED** [review]

File: docs/graphql-api/custom-resolvers.md [hunk 2]
```diff
@@ -1,15 +1,16 @@
...
-sidebar_position: 10
+sidebar_position: 30
 title: Custom API extensions
+description: Extend the API with custom resolvers
 ---
 
 # Custom GraphQL resolvers
 
-One can extend the GraphQL API generated by the OpenReader with custom queries. To do that, one can define GraphQL [query resolvers](https://www.apollographql.com/tutorial/resolvers/) in the designated module `src/server-extension/resolvers`. Note that all resolver classes must be exported.
+One can extend the GraphQL API generated by OpenReader with custom queries. To do that, one can define GraphQL [query resolvers](https://www.apollographql.com/docs/apollo-server/data/resolvers/) in the designated module `src/server-extension/resolvers`. Note that all resolver classes must be exported.
```
[the remark this answers is not in the document]
Comment (abernatskiy) [reply docs/graphql-api/custom-resolvers.md]: Fixed in https://github.com/subsquid/docs/pull/167/commits/7e491ff7b63f33264d74d5a137860b7000ed2e80

File: docs/graphql-api/custom-resolvers.md [hunk 3]
```diff
@@ -1,15 +1,16 @@
...
 
-A custom resolver should import [TypeGraphQL](https://typegraphql.com/resolvers.html#) types and use the annotations provided by the library to define the query arguments and return types.
+A custom resolver should import [TypeGraphQL](https://typegraphql.com/docs/resolvers.html) types and use annotations provided by the library to define query arguments and return types.
 
-Custom resolvers are normally used in combination with [TypeORM EntityManager](https://orkhan.gitbook.io/typeorm/entity-manager-api) for accessing the API server target database. It is automatically injected when defined as a single constructor argument of the resolver. 
+Custom resolvers are normally used in combination with [TypeORM EntityManager](https://orkhan.gitbook.io/typeorm/docs/entity-manager-api) for accessing the API server target database. It is automatically injected when defined as a single constructor argument of the resolver.
```
[the remark this answers is not in the document]
Comment (abernatskiy) [reply docs/graphql-api/custom-resolvers.md]: Fixed in https://github.com/subsquid/docs/pull/167/commits/cc7499df6f6e61b7069120bfb1123c597d8fa980. At some point soon I'll write a script that checks if any external links return anything besides http 200.

File: docs/graphql-api/subscriptions.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (abernatskiy) [reply docs/graphql-api/subscriptions.md]: https://github.com/subsquid/docs/pull/167/commits/07fbc8481826b3018212b7a6154c41fe847ac302

**Pull request merged**